app/controllers/resorts_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ResortsController < ApplicationController
+  before_action :authenticate_user
+
+  def index
+    @resorts = Resort.all
+    render json: @resorts
+  end
+
+  def show
+    @resort = Resort.find(params[:id])
+    render json: { resort: @resort }
+  end
+
+  def create
+    @resort = Resort.new(resort_params)
+    if @resort.save
+      render json: { message: 'resorted created', resort: @resort }, status: :created
```

Comment:
- [ ] [OPTIONAL] Great job, it would be better to also look out for spelling mistakes. Instead of `resorted created`, it would be better to have `resort created`, or `resort created successfully`

Reply:
You right, thanks @VSeray missed it